### PR TITLE
Slim 528 cucumber check response via dbs

### PR DESCRIPTION
**Pull request by robinbakkeruscgi**

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Integration-Tests?pullRequest=27) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/CucumberConstants.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.alliander.osgp.platform.cucumber.support;
```
Comment (hannek24) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/CucumberConstants.java]: missing is the license header

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifier.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package com.alliander.osgp.platform.cucumber.support;
```
Comment (hannek24) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifier.java]: missing is the license header

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/CucumberConstants.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+public interface CucumberConstants {
+
+    String CORRELATION_UID = "CorrelationUid";
```
Comment (hannek24) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/CucumberConstants.java]: Why don't prefix with static?

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifierImpl.java [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+            LOGGER.error(e.getMessage());;
+            System.exit(1);
+        } catch (SQLException e) {
+            LOGGER.error(e.getMessage());;
+            System.exit(2);
+        }
+        return connection;
+    }
+
+    //
```
Comment (hannek24) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/ResponseNotifierImpl.java]: this commented out code can now be removed

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/TestCaseRunner.java [hunk 5]
```diff
@@ -23,8 +26,13 @@
 import com.eviware.soapui.support.SoapUIException;
 
 @Component
-public class TestCaseRunner {
+public class TestCaseRunner implements CucumberConstants {
```
Comment (hannek24) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/support/TestCaseRunner.java]: CucumberConstants  is not used

File: integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/test/ResponseNotifierTest.java [hunk 6]
```diff
@@ -0,0 +1,29 @@
+package com.alliander.osgp.platform.cucumber.test;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+
+import com.alliander.osgp.platform.cucumber.support.ResponseNotifier;
+import com.alliander.osgp.platform.cucumber.support.ResponseNotifierImpl;
+
+public class ResponseNotifierTest {
+
```
Comment (hannek24) [inline integration-tests-cucumber/src/test/java/com/alliander/osgp/platform/cucumber/test/ResponseNotifierTest.java]: The tests in this class are ignored. Can the class be removed?

**Pull request merged**